src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -40,12 +46,25 @@ import { AccountSegment } from './AccountSegement'
 
 export type AccountDropdownProps = {
   assetId: AssetId
-  buttonProps?: ButtonProps
   onChange: (accountId: AccountId) => void
+  accountId?: AccountId
+  // Auto-selects the account with the highest balance, and sorts the account list descending by balance
+  autoSelectHighestBalance?: boolean
+  // Prevents accounts in the dropdown from being selected
+  disableSelection?: boolean
+  buttonProps?: ButtonProps
+  listProps?: MenuItemOptionProps
 }
 
-export const AccountDropdown: React.FC<AccountDropdownProps> = props => {
-  const { assetId, buttonProps, onChange } = props
+export const AccountDropdown: FC<AccountDropdownProps> = ({
+  assetId,
+  buttonProps,
+  onChange: handleChange,
+  disableSelection,
+  accountId: accountIdFromArgs,
```

Comment:
Since this is renamed to avoid shadowing, I'm wondering whether or not we want to rename this at props-level directly, perhaps `defaultAccountId` or `selectedAccountId` to keep things declarative?

Reply:
`defaultAccountId` seems sane to me, I'll make that change.

Reply:
Actioned in https://github.com/shapeshift/web/pull/2684/commits/2221f503e51303bd42014f238ae18f30db489356